src/shapes/metrics_path.cpp

```diff
@@ -229,16 +229,13 @@ void MetricsPath::trim(float startLength, float endLength, bool moveTo, RenderPa
             const PathPart& part = m_Parts[i];
             switch (part.type) {
                 case PathPart::line: {
-                    const Vec2D& point = m_TransformedPoints[part.offset];
-                    result->lineTo(point[0], point[1]);
+                    result->line(m_TransformedPoints[part.offset]);
```

Comment:
Why is the vec2d overload not also called "lineTo"?

Reply:
lineTo is virtual, and didn't want to "shadow" with one virtual and one not with the same name.